README.md

```diff
@@ -13,6 +13,8 @@ In order to install, run:
 make install
 ```
 
+If this process fails because it can't find `.env.test`, run `cp .env .env.test` to create the test environment file.
```

Comment:
Thanks for documenting this! And sorry you had to stumble on it.

Reply:
It's fine. I'm probably going to catch a lot of little things like this as I touch new parts of the codebase.